packages/schema/src/res/stdlib.zmodel

```diff
@@ -633,7 +633,7 @@ attribute @lte(_ value: Int, _ message: String?) @@@targetField([IntField, Float
 /**
  * Validates the entity with a complex condition.
  */
-attribute @@validate(_ value: Boolean, _ message: String?) @@@validation
+attribute @@validate(_ value: Boolean, _ message: String?, _ path: FieldReference?) @@@validation
```

Comment:
I think this parameter should probably be of `String[]` type instead. The `FieldReference` is supposed to refer to a field in the current model (although this is not strictly checked yet); however, in the case of the validation issue path, maybe you'll want to refer to fields in the parent models?

Reply:
agree, this will add more flexibility